core/src/main/groovy/noe/ews/utils/EwsUtils.groovy

```diff
@@ -129,7 +129,9 @@ class EwsUtils extends InstallerUtils {
     } catch (FileNotFoundException e) {
       try {
         loadEwsZipNamesInCompatibilityMode()
-      } catch (FileNotFoundException e) {
+      } catch (FileNotFoundException ex) {
+        log.debug(e.message)
```

Comment:
this log message should be in the `try` statement above

Reply:
I think it's correct. Our concept here breaks some best practices (from my POV), when we use exceptions for controlling the execution (we are not using it for "special unexpected" cases.